### PR TITLE
Sidebar menu for organization management

### PR DESCRIPTION
**Pull request by dinever**
This PR adds a sidebar menu on the classroom settings page.

**Preview:**

screen shot 2016-06-16 at 12 35 06 am

screen shot 2016-06-16 at 12 35 18 am

Please feel free to leave any comment or feedback on this PR. 🍻 

/cc @tarebyte @jasonlong

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -162,6 +164,26 @@
   add_index "repo_accesses", ["organization_id"], name: "index_repo_accesses_on_organization_id", using: :btree
   add_index "repo_accesses", ["user_id"], name: "index_repo_accesses_on_user_id", using: :btree
 
+  create_table "student_identifier_types", force: :cascade do |t|
```
Comment (cyhsutw) [inline db/schema.rb]: @dinever Looks like we don't have this table at this time, we might need to remove this

Comment (dinever) [inline db/schema.rb]: @cyhsutw Just removed it. Thanks!

File: app/views/organizations/invitation.html.erb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+  <div class="columns">
+    <div class="one-fourth column">
+      <%= render partial: 'organizations/side_menu' %>
+    </div>
+    <div class="three-fourths column">
+      <%= render partial: 'organizations/invite_users' %>
+    </div>
+  </div>
+</div>
```
Comment (tarebyte) [inline app/views/organizations/invitation.html.erb]: Line ending here

**Comment by tarebyte** [comment]
@dinever thanks for doing this ✨ 

Can we something like this 

screen shot 2016-06-16 at 08 53 04

To the top of the navigation list. Something like `Classroom settings`

**Comment by dinever** [comment]
screen shot 2016-06-16 at 4 20 19 pm

Hi @tarebyte,

Just added the menu heading ⬆️ , thanks!

**Pull request merged**